apps/tx-builder/src/components/Dashboard.tsx

```diff
@@ -69,6 +70,35 @@ const Dashboard = () => {
     [services.web3],
   );
 
+  const handleAddTransaction = useCallback(
+    (tx: ProposedTransaction) => {
+      setTransactions([...transactions, tx]);
+    },
+    [transactions],
+  );
+
+  const handleRemoveTransaction = useCallback(
+    (index: number) => {
+      const newTxs = transactions.slice();
+      newTxs.splice(index, 1);
+      setTransactions(newTxs);
+    },
+    [transactions],
+  );
+
+  const handleSubmitTransactions = useCallback(async () => {
+    if (!transactions.length) {
+      return;
+    }
+
+    try {
+      await sdk.txs.send({ txs: transactions.map((d) => d.raw) }).catch(console.error);
```

Comment:
As convention we use full names even for this cases. I supose this is a transaction but as I didn't check in deep is not clear to me with only `d`

Don't worry to use longer names

Reply:
I was moving previous code around and didn't see this. Done!!